docs/developer-guide/serialization.md

```diff
@@ -32,16 +32,75 @@ Serialization Formats
 
 ksqlDB supports these serialization formats:
 
--   `DELIMITED` supports comma separated values. See [DELIMITED](#delimited) below.
--   `JSON` and `JSON_SR` support JSON values. See [JSON](#json) below.
--   `AVRO` supports AVRO serialized values. See [AVRO](#avro) below.
--   `KAFKA` supports primitives serialized using the standard Kafka
-    serializers. See [KAFKA](#kafka) below.
--   `PROTOBUF` supports Protocol Buffers. See [Protobuf](#protobuf) below.
+-   [`NONE`](#none) used to indicate the data should not be derialized.
```

Comment:
```suggestion
-   [`NONE`](#none) used to indicate the data should not be deserialized.
```

Reply:
Also, can we move this to the end of the list? Seems confusing to have it as the first item (before the actual formats).

Reply:
I deliberately put it at the top of the list as it's a 'special'  format.

Reply:
I realize but it still feels a bit jarring to have it listed first since it's usage will be relatively uncommon. If the goal is to call attention to the format as being different from the others, maybe we can split this list into two: one for "regular formats" and one for "special formats" or something similalr.